www/about.html

```diff
@@ -24,7 +24,10 @@
     </header>
 
     <main>
-      <a href="/">← Return to the tracker</a>
+      <a href="/">
+        <span aria-hidden="true">←</span>
```

Comment:
should we maybe do this with a css pseudo `:before` thingy? I dunno, just wondering what’s better